### PR TITLE
mobile responsive FAQ title & card

### PR DESCRIPTION
**Pull request by Henry-Graves**
# Describe your changes
- fix mobile text wrapping issues seen here:

image

image

# Issue ticket number and link
NA

### CODE REVIEW

**Comment by Henry-Graves** [comment]
those screenshots are absolutely massive 😭

File: src/pages/FAQ.tsx [hunk 1]
```diff
@@ -121,7 +120,7 @@ export const EditFAQ = () => {
 
   return (
     <VStack alignItems='center' space={2}>
-      <Text variant='header' bold fontSize='3xl'>Frequently Asked Questions</Text>
+      <Text variant='header' bold fontSize='3xl' width='100%'>Frequently Asked Questions</Text>
```
Comment (AlexRutledge1) [inline src/pages/FAQ.tsx]: are these all the props that are needed?

**Review by AlexRutledge1 — APPROVED** [review]
on second thought, send it brother

File: src/pages/FAQ.tsx (hunk 1 above)
Comment (Henry-Graves) [reply src/pages/FAQ.tsx]: i forgor to undo that one, we don't need it.
but yeah line 42 has all the changes we need, the extra ones in the DM were unnecessary

**Review by AlexRutledge1 — APPROVED** [review]
gamer moment

**Pull request merged**